src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java

```diff
@@ -561,33 +586,43 @@ public Set<ModuleReference> findAll() {
         return finder;
     }
 
+    private static String getJlinkRuntimeVersion() {
```

Comment:
Maybe rename to getCurrentRuntimeVersion as that makes it clearer what t his method returns.